autowarefoundation/autoware.universe
# feat(autoware_default_adapi): release adapi v1.6.0

## Description

Release AD API v1.6.0. See release notes for details.

## Related links

https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-interfaces/ad-api/release/

## How was this PR tested?

Door API
1. Launch planning simulation
2. Add DoorPanel
3. Check the doors can be opened/closed in stop mode
4. Check the doors cannot be opened/closed in other modes.

Route API
1. Launch planning simulation
2. Set initialpose and goal
3. Change to autonomous mode
4. Check the route cannot be cleared in autonomous mode
5. Change to stop mode
6. Check the route can be cleared in stop mode

## Notes for reviewers

None.

## Interface changes

None.



## Effects on system behavior

None.
